Edits/azure-service-bus-management-connector-config-topics.adoc

```diff
@@ -0,0 +1,8 @@
+= Azure Service Bus Management 1.0 Additional Configuration Information - Mule 4
+
+Anypoint Connector for Azure Service Bus Management (Azure Service Bus Management Connector) cannot perform a connectivity test at startup using the `Test Connection` button in the global element of the connector configuration when you restrict access to your resources and you have a security policy with permissions at the resource level only. This scenario occurs because the security policy targets the root level of the namespace, which might be forbidden due to the customized policy applied to the shared access key.
+
```

Comment:
I think this is a good idea and will discuss this with the team when we revisit the template. I am not sure we can always move the information in the topic to Troubleshooting, but it might work for this case.